src/main/scala/cromwell/engine/backend/BackendCall.scala

```diff
@@ -105,18 +117,6 @@ trait BackendCall {
   def instantiateCommand: Try[String]
 
   /**
-    * Inputs to the call.  For example, if a call's task specifies a command like this:
-    *
-    * File some_dir
-    * command {
-    *   ls ${some_dir}
-    * }
-    *
-    * Then locallyQualifiedInputs could be Map("some_dir" -> WdlFile("/some/path"))
-    */
-  def locallyQualifiedInputs: CallInputs
-
-  /**
    * Implementation of the WDL Standard Library Functions, used to evaluate WdlExpressions
    */
   def engineFunctions: WdlStandardLibraryFunctions
```

Comment:
I thought that might work but this reference is actually to `CallEngineFunctions` and not the `WorkflowEngineFunctions` that are in the `WorkflowDescriptor`.  I'll change the type here to be more explicit.